doc/compressed-dataset-json1-1.md

```diff
@@ -0,0 +1,186 @@
+# CDISC Compressed Dataset-JSON v1.1 Specification (DSJC)
+
+<table title="Version information">
+  <colgroup>
+    <col class="first">
+    <col>
+  </colgroup>
+  <tbody>
+    <tr>
+      <th>Title</th>
+      <td>CDISC Compressed Dataset-JSON Specification</td>
+    </tr>
+    <tr>
+      <th>Version</th>
+      <td>1.1</td>
+    </tr>
+    <tr>
+      <th>Prepared by</th>
+      <td>CDISC Data Exchange Standards Team</td>
+    </tr>
+    <tr>
+      <th>Notes to Readers</th>
+      <td>This is the specification for Version 1.0 of CDISC Compressed Dataset-JSON.</td>
+    </tr>
+    <tr>
+      <th colspan="1">Revision History</th>
+      <td colspan="1">
+        <table title="Revision  history">
+          <tbody>
+            <tr>
+              <th>Date</th>
+              <th>Version</th>
+              <th>Summary of Changes</th>
+            </tr>
+            <tr>
+              <td>2025-03-18</td>
+              <td>1.1</td>
+              <td>Draft</td>
```

Comment:
Given that we are merging this PR into the master branch of the repo, should we add a line with Final version or replace this one?